### PR TITLE
KOGITO-6107 Main job to update quarkus

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-6107

### CODE REVIEW

File: .ci/jenkins/Jenkinsfile.tools.update-quarkus [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+                    currentBuild.displayName = getNewVersion()
+                }
+            }
+        }
+
+        stage('Call update quarkus jobs') {
+            steps {
+                script {
+                    REPOS.each { repo ->
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.tools.update-quarkus]: `REPOS` is not used again, but I guess you follow that approach just to have the list of repositories more clear, isn't it? So up to you whether to commit this suggestion
```suggestion
                    [ 'kogito-runtimes', 'kogito-examples', 'optaplanner', 'optaplanner-quickstarts' ].each { repo ->
```

**Review by Ginxo — APPROVED** [review]
LGTM, just a minor suggestion

File: .ci/jenkins/Jenkinsfile.tools.update-quarkus [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+
+@Library('jenkins-pipeline-shared-libraries')_
+
+REPOS = [ 'kogito-runtimes', 'kogito-examples', 'optaplanner', 'optaplanner-quickstarts' ]
+
+pipeline {
```
Comment (radtriste) [inline .ci/jenkins/Jenkinsfile.tools.update-quarkus]:
```suggestion
pipeline {
```

File: .ci/jenkins/Jenkinsfile.tools.update-quarkus (hunk 1 above)
Comment (radtriste) [reply .ci/jenkins/Jenkinsfile.tools.update-quarkus]: updated

**Pull request merged**